### PR TITLE
[Elixir] v3 config.json

### PR DESCRIPTION
**Pull request by neenjaw**
Skeleton file after following instructions at:
https://github.com/exercism/v3/blob/master/docs/maintainers/migrating-your-config-json-files.md

### CODE REVIEW

**Comment by neenjaw** [comment]
Config meta from https://github.com/exercism/elixir/blob/master/config.json preserved.  Exercises removed as per instructions for later introduction.

**Comment by Cohen-Carlisle** [comment]
Should [foregone](https://github.com/exercism/elixir/blob/master/config.json#L5) be kept, or is foregone itself... foregone?

**Comment by neenjaw** [comment]
Good question, @ihid?

**Comment by Cohen-Carlisle** [comment]
It looks like it should be kept because I found it [here](https://github.com/exercism/v3/blob/master/languages/csharp/config.json#L55) in C#'s v3 json.

**Comment by neenjaw** [comment]
added

File: languages/elixir/config.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  "blurb": "Elixir is a dynamic, functional language designed for building scalable and maintainable applications. Elixir leverages the Erlang VM, known for running low-latency, distributed and fault-tolerant systems, while also being successfully used in web development and the embedded software domain.",
+  "online_editor": {
+    "indent_style": "space",
+    "indent_size": 2
+  },
+  "exercises": {
+    "concept": [],
+    "practice": []
+  },
+  "foregone": []
```
Comment (Cohen-Carlisle) [inline languages/elixir/config.json]:
```suggestion
  "foregone": ["robot-name"]
```

based on https://github.com/exercism/elixir/blob/master/config.json#L6

Comment (neenjaw) [reply languages/elixir/config.json]: Ah, missed that.  Thanks!

**Comment by ErikSchierboom** [comment]
> Should foregone be kept, or is foregone itself... foregone?

Foregone is ... not foregone! :) It should indeed be kept.

**Comment by ErikSchierboom** [comment]
I'll wait for an Elixir reviewer to look at the contents (especially the editor settings).

**Comment by neenjaw** [comment]
I'm happy with it. We can always make a change if needed.

**Pull request merged**